### ISSUE TITLE
'Uncaught ReferenceError: _EmberApplication is not defined'

### ISSUE THREAD

**Issue by txm** [ember-cli/ember-cli#8958]
Existing App with several Engines, each Engine may have slightly different versions

`node 10.16.3` (and 8)
`ember-cli 2.18.2` (and 2.16.0)
`ember-cli-babel` 7.13.0` (and 7.12.0, 7.0.0)

`ember s`

Browser console: `'Uncaught ReferenceError: _EmberApplication is not defined'`

"It’s to do with ‘@ember/application’ module in app.js and if we use Ember.Application.extend that works, so looks like a Babel issue? "

Assuming it's a dependency that's bumped as this same setup previously built/tested/deployed.

---

Output from `ember version --verbose && npm --version && yarn --version`:
```
~/development/oam (hotfix/test-master) $ ember version --verbose && npm --version && yarn --version
DEPRECATION: ember-cli-babel 5.x has been deprecated. Please upgrade to at least ember-cli-babel 6.6. Version 5.2.8 located: oam -> bg-oam-payments -> ember-data -> ember-cli-babel
DEPRECATION: An addon is trying to access project.nodeModulesPath. This is not a reliable way to discover npm modules. Instead, consider doing: require("resolve").sync(something, { basedir: project.root }). Accessed from:   new NPMDependencyVersionChecker (/Users/andrewmcgregor/development/oam/node_modules/ember-data/node_modules/ember-cli-version-checker/src/npm-dependency-version-checker.js:11:33)
DEPRECATION: ember-cli-babel 5.x has been deprecated. Please upgrade to at least ember-cli-babel 6.6. Version 5.2.8 located: oam -> bg-oam-payments -> ember-data -> ember-inflector -> ember-cli-babel
DEPRECATION: ember-cli-babel 5.x has been deprecated. Please upgrade to at least ember-cli-babel 6.6. Version 5.2.8 located: oam -> bg-oam-payments -> ember-data -> ember-runtime-enumerable-includes-polyfill -> ember-cli-babel
DEPRECATION: ember-cli-babel 5.x has been deprecated. Please upgrade to at least ember-cli-babel 6.6. Version 5.2.8 located: oam -> bg-oam-payments -> ember-elsewhere -> ember-cli-babel
DEPRECATION: ember-cli-babel 5.x has been deprecated. Please upgrade to at least ember-cli-babel 6.6. Version 5.2.8 located: oam -> ember-cli-mirage -> ember-cli-babel
DEPRECATION: ember-cli-babel 5.x has been deprecated. Please upgrade to at least ember-cli-babel 6.6. Version 5.2.8 located: oam -> ember-cli-mirage -> ember-get-config -> ember-cli-babel
DEPRECATION: ember-cli-babel 5.x has been deprecated. Please upgrade to at least ember-cli-babel 6.6. Version 5.2.8 located: oam -> ember-cli-mirage-nested -> ember-cli-babel
DEPRECATION: ember-cli-babel 5.x has been deprecated. Please upgrade to at least ember-cli-babel 6.6. Version 5.2.8 located: oam -> serializer-1x -> ember-cli-babel
DEPRECATION: ember-cli-babel 5.x has been deprecated. Please upgrade to at least ember-cli-babel 6.6. Version 5.2.8 located: oam -> ember-feature-flags -> ember-cli-babel
DEPRECATION: ember-cli-babel 5.x has been deprecated. Please upgrade to at least ember-cli-babel 6.6. Version 5.2.8 located: oam -> ember-inflector -> ember-cli-babel
DEPRECATION: ember-cli-babel 5.x has been deprecated. Please upgrade to at least ember-cli-babel 6.6. Version 5.2.8 located: oam -> ember-legacy-views -> ember-cli-babel
DEPRECATION: ember-cli-babel 5.x has been deprecated. Please upgrade to at least ember-cli-babel 6.6. Version 5.2.8 located: oam -> ember-ordinal-indicators -> ember-cli-babel
DEPRECATION: ember-cli-babel 5.x has been deprecated. Please upgrade to at least ember-cli-babel 6.6. Version 5.2.8 located: oam -> ember-sinon -> ember-cli-babel
DEPRECATION: ember-cli-babel 5.x has been deprecated. Please upgrade to at least ember-cli-babel 6.6. Version 5.2.8 located: oam -> ember-validations -> ember-cli-babel
DEPRECATION: ember-cli-babel 5.x has been deprecated. Please upgrade to at least ember-cli-babel 6.6. Version 5.2.8 located: oam -> ember-validations -> ember-getowner-polyfill -> ember-cli-babel
DEPRECATION: ember-cli-babel 5.x has been deprecated. Please upgrade to at least ember-cli-babel 6.6. Version 5.2.8 located: oam -> emberx-select -> ember-cli-babel
DEPRECATION: ember-cli-babel 5.x has been deprecated. Please upgrade to at least ember-cli-babel 6.6. Version 5.2.8 located: oam -> oam-appointments-engine -> ember-validations -> ember-cli-babel
DEPRECATION: ember-cli-babel 5.x has been deprecated. Please upgrade to at least ember-cli-babel 6.6. Version 5.2.8 located: oam -> oam-appointments-engine -> ember-validations -> ember-getowner-polyfill -> ember-cli-babel
DEPRECATION: ember-cli-babel 5.x has been deprecated. Please upgrade to at least ember-cli-babel 6.6. Version 5.2.8 located: oam -> rewards -> ember-data -> ember-cli-babel
DEPRECATION: ember-cli-babel 5.x has been deprecated. Please upgrade to at least ember-cli-babel 6.6. Version 5.2.8 located: oam -> rewards -> ember-data -> ember-inflector -> ember-cli-babel
DEPRECATION: ember-cli-babel 5.x has been deprecated. Please upgrade to at least ember-cli-babel 6.6. Version 5.2.8 located: oam -> rewards -> ember-data -> ember-runtime-enumerable-includes-polyfill -> ember-cli-babel
DEPRECATION: ember-cli-babel 5.x has been deprecated. Please upgrade to at least ember-cli-babel 6.6. Version 5.2.8 located: oam -> rewards -> ember-validations -> ember-cli-babel
DEPRECATION: ember-cli-babel 5.x has been deprecated. Please upgrade to at least ember-cli-babel 6.6. Version 5.2.8 located: oam -> rewards -> ember-validations -> ember-getowner-polyfill -> ember-cli-babel
DEPRECATION: ember-cli-babel 5.x has been deprecated. Please upgrade to at least ember-cli-babel 6.6. Version 5.2.8 located: oam -> smart-appointments-engine -> ember-data -> ember-cli-babel
DEPRECATION: ember-cli-babel 5.x has been deprecated. Please upgrade to at least ember-cli-babel 6.6. Version 5.2.8 located: oam -> smart-appointments-engine -> ember-data -> ember-inflector -> ember-cli-babel
DEPRECATION: ember-cli-babel 5.x has been deprecated. Please upgrade to at least ember-cli-babel 6.6. Version 5.2.8 located: oam -> smart-appointments-engine -> ember-data -> ember-runtime-enumerable-includes-polyfill -> ember-cli-babel
DEPRECATION: ember-cli-babel 5.x has been deprecated. Please upgrade to at least ember-cli-babel 6.6. Version 5.2.8 located: oam -> smart-appointments-engine -> ember-validations -> ember-cli-babel
DEPRECATION: ember-cli-babel 5.x has been deprecated. Please upgrade to at least ember-cli-babel 6.6. Version 5.2.8 located: oam -> smart-appointments-engine -> ember-validations -> ember-getowner-polyfill -> ember-cli-babel
DEPRECATION: ember-cli-babel 5.x has been deprecated. Please upgrade to at least ember-cli-babel 6.6. Version 5.2.8 located: oam -> ember-data -> ember-cli-babel
DEPRECATION: ember-cli-babel 5.x has been deprecated. Please upgrade to at least ember-cli-babel 6.6. Version 5.2.8 located: oam -> ember-data -> ember-inflector -> ember-cli-babel
DEPRECATION: ember-cli-babel 5.x has been deprecated. Please upgrade to at least ember-cli-babel 6.6. Version 5.2.8 located: oam -> ember-data -> ember-runtime-enumerable-includes-polyfill -> ember-cli-babel
WARNING: Node v10.16.3 is not tested against Ember CLI on your platform. We recommend that you use the most-recent "Active LTS" version of Node.js. See https://git.io/v7S5n for details.
ember-cli: 2.18.2
http_parser: 2.8.0
node: 10.16.3
v8: 6.8.275.32-node.54
uv: 1.28.0
zlib: 1.2.11
brotli: 1.0.7
ares: 1.15.0
modules: 64
nghttp2: 1.39.2
napi: 4
openssl: 1.1.1c
icu: 64.2
unicode: 12.1
cldr: 35.1
tz: 2019a
os: darwin x64
6.13.1
-bash: yarn: command not found
```

**Comment by rwjblue** [comment]
I believe we figured this out in discord chat, but for others stumbling across this PR has a good explanation of what is going on https://github.com/babel/ember-cli-babel/pull/284. 

tldr; the `config/targets.js` in use allows IE < 11

**Comment by txm** [comment]
> module.exports = {
>   browsers: [
>     'ie 9',
>     'last 1 Chrome versions',
>     'last 1 Firefox versions',
>     'last 1 Safari versions'
>   ]
> };

**Comment by iamareebjamal** [comment]
Seeing this even if target has `ie 11`
